lib/Alchemy/Phrasea/Controller/Api/V1Controller.php

```diff
@@ -1178,7 +1179,8 @@ public function searchAction(Request $request)
         $searchView = $this->buildSearchView(
             $result,
             $includeResolver->resolve($fractal),
-            $this->resolveSubdefUrlTTL($request)
+            $this->resolveSubdefUrlTTL($request),
+            $request
```

Comment:
The whole request should not be passed to a function that does not need it (for building a response). "buildSearchView()" should not know it's called from an (http)action. Ideally, request should only appear on route handlers "xxxAction()". I know that "request" is sent everywhere (easy), it's not a big deal on private methods that build final response, but here, it's nicer to compute "max_items", "page" into the handler "searchAction()", and then pass them to "buildSearchView()"
Just aesthetic...